### ISSUE TITLE
Exposure to silver

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9855]
**Preferred term label:**
Exposure to silver

**Synonyms**
Occupational exposure, jeweler, miner, colloidal silver

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Argyria

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Would **Occupational exposure to silver** be a better label here to capture the intensity and amount of exposure (i.e., not just polishing the family silverware once a year)

**Comment by MickeySegal** [comment]
That is good.

**Comment by pnrobinson** [comment]
@MickeySegal 
adding as

Occupational silver exposure
HP:0035011
def: A history of exposure to silver that has occured in relation to performance of job duties.
